app/models/verse.rb

```diff
@@ -13,11 +13,22 @@ def self.last_month_winner
 
   def self.previous_month
     # 1.month.ago
-    current_month = Time.now.mon
     if current_month == 1
       return 12
     else
       current_month - 1
     end
   end
+
+  def self.current_month
+    current_month = Time.now.mon
```

Comment:
I don't think we need to assign the variable here since that is all the method is returning.